tezos/new_context/src/gc/worker.rs

```diff
@@ -0,0 +1,183 @@
+use std::{
```

Comment:
@sebastiencs one small thing, here and in all source code files we need the copyright notice comment that we have everywhere else.